tests/system/action/poll/test_vote.py

```diff
@@ -0,0 +1,382 @@
+                    "data": [{"id": 1, "user_id": 1, "value": "N"}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        vote = self.get_model("vote/1")
+        assert vote.get("value") == "N"
+        assert vote.get("option_id") == 11
+        assert vote.get("weight") == "1.000000"
+        assert vote.get("meeting_id") == 113
+        assert vote.get("user_id") == 1
+        option = self.get_model("option/11")
+        assert option.get("vote_ids") == [1]
+        assert option.get("yes") == "0.000000"
+        assert option.get("no") == "1.000000"
+        assert option.get("abstain") == "0.000000"
+        user = self.get_model("user/1")
+        assert user.get("vote_$_ids") == ["113"]
+        assert user.get("vote_$113_ids") == [1]
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "poll.vote",
+                    "data": [{"id": 1, "user_id": 2, "value": "Y"}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_not_exists("vote/2")
+        option = self.get_model("option/11")
+        assert option.get("vote_ids") == [1]
+        user = self.get_model("user/1")
+        assert user.get("vote_$_ids") == ["113"]
+        assert user.get("vote_$113_ids") == [1]
+
+    def test_vote_schema_problems(self) -> None:
+        self.create_model("group/1", {"user_ids": [1]})
+        self.create_model(
+            "poll/1", {"title": "my test poll", "entitled_group_ids": [1]}
+        )
+        self.create_model("meeting/113", {"name": "my meeting"})
+        self.update_model("user/1", {"is_present_in_meeting_ids": [113]})
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "poll.vote",
+                    "data": [{"id": 1, "user_id": 1, "value": "X"}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 400)
+        assert (
+            "data.value must be valid by one of anyOf definition"
+            in response.data.decode()
+        )
+
+    def test_vote_for_analog_type(self) -> None:
+        self.create_model("group/1", {"user_ids": [1]})
+        self.create_model(
+            "poll/1",
+            {"title": "my test poll", "type": "analog", "entitled_group_ids": [1]},
+        )
+        self.create_model("meeting/113", {"name": "my meeting"})
+        self.update_model("user/1", {"is_present_in_meeting_ids": [113]})
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "poll.vote",
+                    "data": [{"id": 1, "user_id": 1, "value": "Y"}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 400)
+        assert "poll.vote is not allowed for analog voting." in response.data.decode()
```

Comment:
See above, this should be allowed

Reply:
This is correct, we have the option.update action for updating analog polls.